lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.mjs

```diff
@@ -142,6 +142,78 @@ testRule({
 			endLine: 1,
 			endColumn: 138,
 		},
+		{
+			code: `a {
+				border-top-width: 1px;
+				border-right-width: 2px;
+				border-bottom-width: 3px;
+				border-left-width: 4px;
+				border-top-color: red;
+				border-right-color: green;
+				border-bottom-color: blue;
+				border-left-color: yellow;
+				border-top-style: dotted;
+				border-right-style: solid;
+				border-bottom-style: dashed;
+				border-left-style: double;
+			}`,
+			fixed: `a {
+				border-width: 1px 2px 3px 4px;
+				border-color: red green blue yellow;
+				border-style: dotted solid dashed double;
+			}`,
+			warnings: [
+				{
+					column: 5,
+					endColumn: 22,
+					endLine: 5,
+					line: 5,
```

Comment:
[suggest] The order of `line`, `column`, `endLine`, and `endColumn` is more consistent:

```suggestion
					line: 5,
					column: 5,
					endLine: 5,
					endColumn: 22,
```

Reply:
2c3d386 doesn't resolve it. Please see the order in the same file: https://github.com/stylelint/stylelint/blob/2c3d38652c3b9a8be6abf829ab02c45ac5bcd8f7/lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.mjs#L95-L98

Reply:
Sorry about that I must read it incorrectly.